lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -82,7 +83,13 @@ If (Test-Path -LiteralPath $path -PathType Leaf) {
         $result.output = "Run pester test in the file: $path"
```

Comment:
Shouldn't you add something in the checkmode to reflect the changes too?

Reply:
How about `Run pester test in the file: $path with parameterlist if specified` ?

Reply:
Maybe add the parameter names to the output

Reply:
make sense